catalogd/cmd/catalogd/main.go

```diff
@@ -63,6 +63,7 @@ import (
 	"github.com/operator-framework/operator-controller/catalogd/internal/storage"
 	"github.com/operator-framework/operator-controller/catalogd/internal/version"
 	"github.com/operator-framework/operator-controller/catalogd/internal/webhook"
+	"github.com/operator-framework/operator-controller/internal/util"
```

Comment:
would it be possible to [avoid introducing `util` and have the new functions located in a more meaningful place(s)](https://100go.co/#creating-utility-packages-13)?

Reply:
Do you have some suggestions? As always, naming is a PITA... I'm struggling to find a good name for a package that contains common shared utility functions used by various other packages in the project XD should we just call the package `fs`?

Reply:
if the functions / utils deal with filesystem then `fs` makes sense as package name to me